lib/parser.ts

```diff
@@ -155,7 +153,7 @@ export class Parser {
 
     try {
       return rawActivities.map((rawActivity) =>
-        this.isTradeActivity(rawActivity)
+        rawActivity.activity_type === 'FILL'
```

Comment:
Now that TypeScript knows about the `activity_type` possibilities in `RawTradingActivity` and `RawNonTradingActivity`, it can be smart about ternaries like this.